site_reliability_guardian_sample/README.md

```diff
@@ -0,0 +1,20 @@
+
+# Site Reliability Guardian
+
+This sample provides the configuration for a guardian in the [Site Reliability Guardian](https://www.dynatrace.com/support/help/platform-modules/cloud-automation/site-reliability-guardian) and an automation workflow leveraging [Workflows](https://www.dynatrace.com/support/help/platform-modules/cloud-automation/workflows).
+
+## Prerequisites
+
+- Monaco >`v2.6.0`
+- [Install Site Reliability Guardian](https://www.dynatrace.com/support/help/platform-modules/cloud-automation/site-reliability-guardian#install-update-or-uninstall)
```

Comment:
Please mention that this sample requires a Dynatrace Platform environment and oauth token with the correct permissions to configure the SRG app's settings. 

You can link to the monaco docs on oauth authentication: https://www.dynatrace.com/support/help/manage/configuration-as-code/configuration#oauth

Reply:
Valid point. I added the OAuth client.

Reply:
Perfect, thanks